src/redux/configureStore.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable import/extensions */
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import { logger } from 'redux-logger';
+import booksReducer from './books/books.js';
```

Comment:
- [ optional ] The `.js` extension from `books.js` is causing the Linter errors. You can import with `import booksReducer from './books/books';` so you don't have to deal with the Linter errors

Reply:
Oh Yeah! Sorry for that, I was trying to make Heroku deployment work so I was trying all types of changes hahaha. I'll change it